src/components/Editor/TiptapEditor.vue

```diff
@@ -157,6 +157,29 @@ export default {
 
 .editor__content {
   margin-top: 0.5rem;
+  p {
+    font-size: 18px;
+    line-height: 30px;
+  }
```

Comment:
UI/UX reported that text in paragraph isn't #000000 (black).
![圖片](https://user-images.githubusercontent.com/32424677/140596677-4bffc360-ddea-4f59-9c8a-78bac44bac8e.png)
```suggestion
  p {
    font-size: 18px;
    line-height: 30px;
    color: #000000;
  }
```

Reply:
Thanks!